test/Microsoft.AspNetCore.Sockets.Tests/Microsoft.AspNetCore.Sockets.Tests.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
+    
+    <RuntimeIdentifier Condition="'$(TargetFramework)' != 'netcoreapp2.0'">win7-x64</RuntimeIdentifier>
```

Comment:
@natemcmaster should this be in all of our test projects aka Directory.Build.props

Reply:
Do we need it now? Why didn't we need it before?

Reply:
Because we never started the server before in this project. The other tests just build the pipeline.

Reply:
The preferred way to do this is with HasRuntimeOutput. Example:

https://github.com/aspnet/SignalR/blob/d3ae3abe2acb97bf69143c85de9d4843e4d54e10/test/Microsoft.AspNetCore.SignalR.Tests/Microsoft.AspNetCore.SignalR.Tests.csproj#L7-L11

This should be set automatically in future versions of Microsoft.NET.Test.Sdk